src/modules/shared/components/Menu/Menu.tsx

```diff
@@ -0,0 +1,129 @@
+import { ReactNode, useState } from 'react';
+import { useCopyToClipboard } from 'react-use';
+
+import classNames from 'classnames';
+
+import { ReactComponent as CopyIcon } from '../../assets/icons/copyIconOutlined.svg';
+import { ReactComponent as CardIcon } from '../../assets/icons/creditCardOutlined.svg';
+import { ReactComponent as HelpIcon } from '../../assets/icons/helpCircleOutlined.svg';
+import { ReactComponent as ImageIcon } from '../../assets/icons/imageOutlined.svg';
+import { ReactComponent as LogoutIcon } from '../../assets/icons/logoutOutlined.svg';
+import { ReactComponent as SettingsIcon } from '../../assets/icons/settingsOutlined.svg';
+import { ReactComponent as UserIcon } from '../../assets/icons/userOutlined.svg';
+import { PixwayAppRoutes } from '../../enums/PixwayAppRoutes';
+import { useProfile } from '../../hooks';
+import { Link } from '../Link';
+
+interface MenuProps {
+  tabs?: TabsProps[];
+  className?: string;
+}
+
+interface TabsProps {
+  title: string;
+  icon: ReactNode;
+  link: string;
+}
```

Comment:
A nomenclatura não está muito correta
Props em react são argumentos passados para componentes.
```suggestion
interface TabConfig {
  title: string;
  icon: ReactNode;
  link: string;
}
```